spec/index.bs

```diff
@@ -410,6 +410,23 @@ The [=subresources=] of textures included in the views provided to
 are considered to have {{GPUTextureUsage/OUTPUT_ATTACHMENT}}
 for the [=usage scope=] of this render pass.
 
+The <dfn dfn>physical size</dfn> of a {{GPUTexture}} [=subresource=] is the dimension of the {{GPUTexture}}
```

Comment:
In a way I think we should define what a texel block is, with a definition that explains that regular format have a texel block size of 1. Then we can define pixel-based formats (1x1 blocks), and block-based formats (other block sizes). The discussion of compressed formats would only be in examples.

Then a lot of the discussion can only talk about block sizes in general, and have notes for what happens for pixel-based formats (i.e. no extra padding).

Basically the spec would be written as if all formats are block-based, but would explain what happens in the usual case when the block size is 1x1.

Reply:
The definition of "texel block" is in the Chapter "Texture Formats", which I think is a better place for it. What do you think?

Reply:
It's probably okay if the definition of texel block is in Texture Formats but we reference it from here. The important comment here, though, is about rewording everything so that there's no "pixel-based" vs "block-based" format distinction. It allows us to write a lot of spec text without mentioning the difference at all, which simplifies it and makes it more readable.